packages/foundations-forms/src/forms.pcss

```diff
@@ -306,6 +310,18 @@ input[type="radio"],
   touch-action: manipulation;
 }
 
+.coop-form__choice .coop-form__hint {
+  margin-top: 0;
+  padding-left: calc(
+    var(--form-choice-width) - var(--form-choice-stroke-width) +
```

Comment:
Minor thing but could this be on a single line, it's making me twitch 😆

Reply:
@gbbns Ha, tell me about it! This is auto-formatted by our `stylelint` rules. Can we adjust them??